base/docs/helpdb/Base.jl

```diff
@@ -1278,6 +1278,10 @@ hash
     read(stream::IO, T)
 
 Read a single value of type `T` from `stream`, in canonical binary representation.
+
+    read(stream::IO, String)
+
+Read the entirety of `stream`, as a String.
```

Comment:
For consistency with the other docstring you added, I'd remove the comma and lowercase string. If you leave it uppercase as in the type, it should have code formatting.